packages/components/src/components/Table/Table.tsx

```diff
@@ -321,11 +322,16 @@ export const TableRow = forwardRef<HTMLTableRowElement, TableRowProps>(
     ) => {
         const isInteractive = onClick !== undefined && !disabled;
 
-        const handleClick = () => {
+        const handleClick = (event?: MouseEvent<HTMLTableRowElement>) => {
             if (disabled) {
                 return;
             }
 
+            const clickedCell = event?.target instanceof Element ? event.target.closest('td') : null;
```

Comment:
Why not use event.target directly and check if it is a button or a link? As it is implemented now we'd prevent handling of clicks in the whitespace of a cell just because the cell contains any of those things.

Reply:
Yes you're right gg.